app/src/main/java/com/jerboa/ui/components/post/PostActivity.kt

```diff
@@ -113,9 +135,13 @@ fun PostActivity(
     val unExpandedComments = remember { mutableStateListOf<Int>() }
     val commentsWithToggledActionBar = remember { mutableStateListOf<Int>() }
     var showSortOptions by remember { mutableStateOf(false) }
+    val focusRequester = remember { FocusRequester() }
 
     val listState = rememberLazyListState()
+    var lazyListIndexTracker: Int
```

Comment:
This should probably be in the remembered state as well.

Reply:
The problem I had with this is that this counter [needed to be reset](https://github.com/dessalines/jerboa/pull/749/files/573a154d6cb52b624803dc5530a91501d168576a#diff-9958f04ae114c8444c9e781405889c7fab0b5465ca616011fcfb047a243c356bR246) every time the comments were composed, and the comments are getting recomposed all the time when scrolling. Without resetting it there, the counter would just keep rising. I actually needed to clear `parentListStateIndexes` for the same reason, it would just keep adding the parent indexes forever while scrolling.

So if I define it like `var lazyListIndexTracker = remember { 2 }`, I get a warning `Variable 'lazyListIndexTracker' initializer is redundant`

I'm very new to Kotlin and Jetpack Compose, so I might be missing a better way to do this

Reply:
Ah, if it needs to be reset every time then no need.